js/PivotReport.PivotTable.js

```diff
@@ -1,5 +1,7 @@
 /* globals _, CRM, moment, ts */
 
+var DEFAULT_DATE_FORMAT = 'YYYY-MM-DD';
```

Comment:
Dont make it global, please define inside  `CRM.PivotReport.PivotTable = (function ($) {`